File: Coding/coding modul 2/PRAK205-2410817110007-MUHAMMAD-HAMKA-AKBARI.py
import math 

tinggi, miring = map(int,input().split())

alas = math.sqrt(pow(miring,2) - pow(tinggi,2))
keliling = alas + tinggi + miring
luas = 1/2 * alas * tinggi

print(f"Alas = {alas:.0f} cm")
print(f"Tinggi = {tinggi:.0f} cm")
print(f"Keliling = {keliling:.0f} cm")
print(f"Luas = {luas:.0f} cm^2\n")

tinggi = int(input())
miring = int(input())

alas = math.sqrt(pow(miring,2) - pow(tinggi,2))
keliling = alas + tinggi + miring
luas = 1/2 * alas * tinggi

print(f"Alas = {alas:.0f} cm")
print(f"Tinggi = {tinggi:.0f} cm")
print(f"Keliling = {keliling:.0f} cm")
print(f"Luas = {luas:.0f} cm^2\n")

banyak_variable = 2
inputs = []

while len(inputs) < banyak_variable:
    line = input()

    new_line = line.split()

    for value in new_line:
        inputs.append(float(value))

tinggi, miring = inputs

alas = math.sqrt(pow(miring,2) - pow(tinggi,2))
keliling = alas + tinggi + miring
luas = 1/2 * alas * tinggi

print(f"\nAlas = {alas:.0f} cm")
print(f"Tinggi = {tinggi:.0f} cm")
print(f"Keliling = {keliling:.0f} cm")
print(f"Luas = {luas:.0f} cm^2\n")

File: coding modul 2/PRAK205-2410817110007-MUHAMMAD-HAMKA-AKBARI.c
#include <stdio.h>
#include <math.h>

int main()
{
    int tinggi, miring, alas, keliling, luas;

    scanf("%d %d", &tinggi, &miring);

    alas = sqrt(pow(miring,2) - pow(tinggi,2));
    keliling = tinggi + miring + alas;
    luas = 0.5 * alas * tinggi;

    printf("\nAlas : %d cm", alas);
    printf("\nTinggi : %d cm", tinggi);
    printf("\nKeliling : %d cm", keliling);
    printf("\nLuas : %d cm^2", luas);

    return 0;
}